include/industry_standard/spdm.h

```diff
@@ -19,6 +19,7 @@
 #define SPDM_MAX_SLOT_COUNT 8
 #define SPDM_MAX_OPAQUE_DATA_SIZE 1024
 
+#define MAX_ROOT_CERT_SUPPORT 10
```

Comment:
please move this to config.h, and add LIBSPDM_ prefix.

Reply:
Yes, I have changed it. Thanks.